assets/templates/partials/census/description.tmpl

```diff
@@ -0,0 +1,21 @@
+<h2 class="u-fw-b">{{ localise "Summary" .Language 1 }}</h2>
```

Comment:
An observation, I think the plan is for this component to be used in a few places and as its use increases it will be useful to be a partial where you can instantiate by sending in parameters.

Reply:
Makes sense. I think then we will need to create a new model that makes this feasible, particularly if there's a chance of there being multiple of these sections. Are we anticipating that the collapsible component will be something that the publishing team can populate themselves?